mod/htmlawed/classes/HtmlawedPlugin/ValidateInputHook.php

```diff
@@ -0,0 +1,141 @@
+<?php
+namespace HtmlawedPlugin;
```

Comment:
`Elgg\Mod\` for plugins would eliminate chance of conflict with system classes. Creating these directory trees is annoying. We need an elgg-cli.